android/src/main/java/com/swmansion/rnscreens/SearchBarManager.kt

```diff
@@ -131,4 +132,56 @@ class SearchBarManager : ViewGroupManager<SearchBarView>() {
     companion object {
         const val REACT_CLASS = "RNSSearchBar"
     }
+
+    private fun logNotAvailable(propName: String) {
+        Log.w("RN SCREENS", "$propName prop is not available on Android")
+    }
+
+    // NativeCommands
+
+    override fun blur(view: SearchBarView?) {
+        view?.handleBlurJsRequest()
+    }
+
+    override fun focus(view: SearchBarView?) {
+        view?.handleFocusJsRequest()
+    }
+
+    override fun clearText(view: SearchBarView?) {
+        view?.handleClearTextJsRequest()
+    }
+
+    override fun toggleCancelButton(view: SearchBarView?, flag: Boolean) {
+        view?.handleToggleCancelButtonJsRequest(flag)
+    }
+
+    override fun setText(view: SearchBarView?, text: String?) {
+        view?.handleSetTextJsRequest(text)
+    }
+
+    override fun cancelSearch(view: SearchBarView?) {
+        view?.handleFocusJsRequest()
+    }
+
+    // iOS only
+
+    override fun setHideWhenScrolling(view: SearchBarView?, value: Boolean) {
+        logNotAvailable("hideWhenScrolling")
+    }
+
+    override fun setObscureBackground(view: SearchBarView?, value: Boolean) {
+        logNotAvailable("hideNavigationBar")
+    }
+
+    override fun setHideNavigationBar(view: SearchBarView?, value: Boolean) {
+        logNotAvailable("hideNavigationBar")
+    }
+
+    override fun setCancelButtonText(view: SearchBarView?, value: String?) {
+        logNotAvailable("cancelButtonText")
+    }
+
+    override fun setTintColor(view: SearchBarView?, value: Int?) {
+        logNotAvailable("tintColor")
+    }
```

Comment:
Same remarks as to `setPlacement` prop.

Reply:
Yeah, I added to todo and will change that in all places in separate PR.